testapps/ClassLibraryWithPrecompiledViews/project.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "preserveCompilationContext": true,
+    "emitEntryPoint": true
+  },
+  "packOptions": {
+    "files": {
+      "mappings": {
+        "lib/": "obj/precompiled/" 
+      }
+    }
+  },
+  "dependencies": {
+    "Microsoft.AspNetCore.Mvc": "1.1.0-*",
+    "Microsoft.AspNetCore.Mvc.Razor.Precompilation.Design": {
+      "version": "1.0.0-*",
+      "target": "package",
+      "type": "build"
+    }
+  },
+  "tools": {
+    "Microsoft.AspNetCore.Mvc.Razor.Precompilation.Tools": "1.0.0-*"
+  },
+
+  "frameworks": {
+    "netcoreapp1.0": {
```

Comment:
Hmm, so this isn't a real application? What's the point of these tests if we're working around dotnet limitations currently?